deepchecks/nlp/datasets/token_classification/scierc_ner.py

```diff
@@ -0,0 +1,156 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2023 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Dataset containing abstracts from scientific journals and named entity annotations for relevant scientific words.
+
+The data contains 350 samples for the train set and 100 samples for the test set. Each dataset sample is a tokenized
```

Comment:
That's the original size? That's really small for us to be able to demonstrate some thing... no? E.g. embeddings drift